common/djangoapps/util/course.py

```diff
@@ -0,0 +1,31 @@
+"""
+Utility methods related to course
+"""
+import logging
+from django.conf import settings
+
+from opaque_keys.edx.keys import CourseKey
+import re
+
+log = logging.getLogger(__name__)
+
+
+def get_lms_link_for_about_page(course_key):
+    """
+    Returns the url to the course about page from the location tuple.
+    """
+    assert isinstance(course_key, CourseKey)
+
+    if settings.FEATURES.get('ENABLE_MKTG_SITE', False):
+        # Root will be "https://www.edx.org". The complete URL will still not be exactly correct,
+        # but redirects exist from www.edx.org to get to the Drupal course about page URL.
+        about_base = settings.MKTG_URLS.get('ROOT')
+    else:
+        about_base = settings.ENV_TOKENS.get('LMS_BASE')
+
+    # Strip off https:// (or http://) to be consistent with the formatting of LMS_BASE.
+    about_base = re.sub(r"^https?://", "", about_base)
+    return u"https://{about_base_url}/courses/{course_key}/about".format(
```

Comment:
@jcdyer Thanks for handling it.
I would help maintain scheme for `LMS_ROOT_URL`; but for marketing `ROOT` url we still need to strip http/https and replace it with the scheme used by lms scheme or cms scheme depending upon from where the method is being called (cc: @ahsan-ul-haq )

For this, we need to do two things:
1. know from where this common method is being called (i.e. lms or cms)
2. replace marketing url scheme with lms or cms scheme (decision based on first).
for 2, we can get lms scheme from `LMS_ROOT_URL` but (as your other PR states) we donot have any config against cms scheme.

What do u suggest?